draft-ftbs-rats-msg-wrap.md

```diff
@@ -75,7 +75,7 @@ message(s) are carried in the value.
 The other format wraps the value in a CBOR byte string and prepends a
 CBOR tag to convey the type information.
 
-An associated CBOR tag, as well as JWT and CWT claims are also defined to allow embedding these formats into CBOR-based protocols and web tokens.
+This document defines a corresponding CBOR tag, as well as JWT and CWT claims.  These allow embedding the wrapped conceptual messages into CBOR-based protocols and web tokens.
```

Comment:
```suggestion
This document defines a corresponding CBOR tag, as well as JWT and CWT claims.  These allow embedding the wrapped conceptual messages into CBOR-based protocols and structures, e.g., web tokens.
```

Reply:
Ned, the problem with the suggestion is that JWT is not a CBOR-based structure.

The two sentences are structured symmetrically:
* CBOR tag -> CBOR-based protocols
* JWT and CTW claims -> web tokens

Maybe we should add a "respectively" to make the connection more explicit?

Reply:
OK

Reply:
see b288559f5c0e6e506ef81fa8af5bd233756a707e